product/reports/200_Optimization/001_CPU Right Sizing Recommendations.yaml

```diff
@@ -0,0 +1,95 @@
+---
```

Comment:
The naming of the file requires a dash, otherwise it doesn't display the report itself:
![Screenshot from 2019-08-21 15-19-39](https://user-images.githubusercontent.com/649130/63435396-1d8c1880-c427-11e9-97c1-1f51390b45b5.png)

Reply:
> The naming of the file requires a dash, otherwise it doesn't display the report itself

Wrong :-(

The name of the directory requires a dash. Meaning the minimal level of nesting is 2.

Your comment was rather misleading.

Reply:
Sorry, I knew there's a dash required somewhere though :confetti_ball: